cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/table/StructuredTableSchema.java

```diff
@@ -33,22 +35,22 @@
   private final StructuredTableId tableId;
   private final Map<String, FieldType.Type> fields;
   // primary keys have to be ordered as defined in the table schema
-  private final Set<String> primaryKeys;
```

Comment:
what is the reason for making this change?

Reply:
Because we want to compare the prefix of primary keys match the keys provided, changing this to list will make it clearer and easier do the check